docs/goals-canvas.md.txt

```diff
@@ -0,0 +1,26 @@
+| Zoek of raadpleeg  eigenschappen van personen waarmee je persoon kunt aanschrijven in formele en minder formele communicatie (brief, e-mail, dm etc.) | Door te zoeken of raadplegen en/of filteren op naamkenmerken  voor het aanschrijven van persoon | Bovenstaande zoek- en raadpleegopties + optioneel naamfilter waarbij "aanhef", "aanschrijfwijze", "regel voorafgaand aan aanschrijfwijze" en          "gebruik in lopende tekst" en "voorletters" naar wens kunnen worden opgegeven. | (gefilterde) persoon bevat naast bovenstaande gegevens ook voorletters, aanhef, aanschrijfwijze, regel voorafgaand aan aanschrijfwijze en gebruik in lopende tekst. | Personen direct correct formeel en minder formeel aanschrijven of  correct tot hen richten in bijvoorbeeld e-mailverkeer of dm bij het uitvoeren van een wettelijke taak. | [781](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/781){:target="_blank"}, [758](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/758){:target="_blank"}, [736](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/736){:target="_blank"}, [734](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/734){:target="_blank"}, [733](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/733){:target="_blank"}, [732](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/732),  [730](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/730),   [729](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/729),  [727](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/727), {:target="_blank"}, [726](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/726){:target="_blank"}, [725](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/725){:target="_blank"}, [724](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/724){:target="_blank"}, [717](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/717){:target="_blank"}, [713](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/713){:target="_blank"},  [689](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/689){:target="_blank"}, [675](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/675){:target="_blank"}, [674](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/674){:target="_blank"}, [636](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/636){:target="_blank"}, |
```

Comment:
bij what zou ik 'zoek of raadpleeg' weglaten en iets zeggen a la: samengestelde gegevens nodig voor het kunnen aanschrijven in formele en minder formele communicatie (brief, e-mail, dm etc.) van gevonden personen
bij how zou ik iets zeggen als: bij filteren naam van deze kenmerken opgeven in de filter en bij niet-filteren krijgt je het standaard
Wat wordt met naamkenmerken in de input bedoeld? Of bedoel je naam van de kenmerken t.b.v. het aanschrijven van personen

Reply:
met naamkenmerken bedoel ik de kenmerken die bij de naam horen, zoals deze kenmerken.